.github/workflows/algolia/src/uploadToAlgolia.ts

```diff
@@ -1,8 +1,7 @@
-import { generateAlgoliaJson, uploadAlgoliaData } from ".";
+import { syncAlgoliaIndex } from ".";
```

Comment:
Importing from `.` adds a risk of circular imports. Can we `import { syncAlgoliaIndex } from "./syncAlgoliaIndex";